akka-actor-tests/src/test/java/akka/actor/ActorCreationTest.java

```diff
@@ -117,5 +141,25 @@ public void testRejectAbstractActorClass() {
       assertEquals(String.format("Actor class [%s] must not be abstract", H.class.getName()), e.getMessage());
     }
   }
-  
+
+  private static Creator<UntypedActor> createAnonymousCreatorInStaticMethod() {
+    return new Creator<UntypedActor>() {
+      @Override
```

Comment:
This is the weird case - this anonymous class would be marked as `static` because it's inside a static method, which violates: `An anonymous class is always an inner class (8.1.3); it is never static (8.1.1, 8.5.1).`

http://bugs.java.com/bugdatabase/view_bug.do?bug_id=8034044